engines/ultima/ultima8/gumps/gump.h

```diff
@@ -108,23 +108,20 @@ class Gump : public Object {
 	//! \param takefocus If true, set parent's _focusChild to this
 	virtual void                InitGump(Gump *newparent, bool take_focus = true);
 
-	//! Find a gump of the specified type (this or child)
-	//! \param t Type of gump to look for
+	typedef bool (Gump::*FindPredicate)() const;
```

Comment:
Should I consider a Common::UnaryFunction for this instead of member function pointer?

Reply:
I don't have a strong feeling on this either way. Since the usage is fairly limited it may be easier to just do it like this?